src/components/DeleteConfirmationModal/DeleteConfirmationModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FunctionComponent } from 'react';
+import {
+  Button,
+  Checkbox,
+  List,
+  ListItem,
+  Modal,
+  ModalVariant,
+  Text,
+  TextContent,
+  TextVariants
+} from '@patternfly/react-core';
+import useDeleteSatelliteManifest from '../../hooks/useDeleteSatelliteManifest';
+import { Processing } from '../emptyState';
+import './DeleteConfirmationModal.scss';
+
+interface DeleteConfirmationModalProps {
+  handleModalToggle: () => void;
+  isOpen: boolean;
+  name: string;
+  uuid: string;
+}
+
+const DeleteConfirmationModal: FunctionComponent<DeleteConfirmationModalProps> = ({
+  handleModalToggle,
+  isOpen,
+  name,
+  uuid
+}) => {
+  const [checked, setChecked] = React.useState(false);
+  const { isLoading: isDeletingManifest, mutate: deleteManifest } = useDeleteSatelliteManifest();
+
+  const handleCheckbox = (checked: boolean, event: React.ChangeEvent<HTMLInputElement>) => {
+    setChecked(event.target.checked);
+  };
+
+  const closeModal = () => {
+    handleModalToggle();
+    setChecked(false);
+  };
+
+  const handleConfirmation = () => {
+    deleteManifest(uuid);
+    closeModal();
+  };
+
+  const actions = () => {
+    if (isDeletingManifest) {
+      return [];
+    } else {
+      return [
+        <Button key="cancel" variant="link" onClick={closeModal}>
+          NO, CANCEL
+        </Button>,
+        <Button key="confirm" variant="primary" isDisabled={!checked} onClick={handleConfirmation}>
+          YES, DELETE
+        </Button>
+      ];
+    }
+  };
+
+  const content = () => {
```

Comment:
I don't want to add random error messaging into a random location in the UI.  Let's make a note of it in the story you added.  We should have thought to bring this up in grooming.

Also I want to be clear what we are trying to do because as is your story does not specify.  I think we should have a reusable notification system, like that toast one you suggested in the frontend components library.  (Does that make popups in the corner of the screen?  I didn't immediately see a demo of it.)  Alternatively we could copy the dull way our Ruby apps show messages with flash banners in a fixed location at the  top of the page.  We'll probably want to consult with Russ about the options too.